core_backend/tests/api/test_dashboard.py

```diff
@@ -662,13 +662,14 @@ async def content_data(self, asession: AsyncSession) -> AsyncGenerator[None, Non
                 .astype(np.float32)
                 .tolist(),
                 content_title=c["title"],
-                content_text="Test content #{i}",
+                content_text=f"Test content #{_i}",
                 content_metadata={},
                 created_datetime_utc=datetime.now(timezone.utc),
                 updated_datetime_utc=datetime.now(timezone.utc),
                 query_count=c["query_count"],
                 positive_votes=c["positive_votes"],
                 negative_votes=c["negative_votes"],
+                is_archived=True,
```

Comment:
So all the content it is creating is archived already? We only want some to be archived right?

Reply:
Sounds good--will create some content that is archived and some not.